src/components/Header/Tools.tsx

```diff
@@ -150,7 +150,7 @@ const Tools = () => {
     },
     {
       title: intl.formatMessage(messages.supportOptions),
-      url: isITLessEnv ? 'https://redhatgov.servicenowservices.com/css' : 'https://access.redhat.com/support',
+      onClick: () => (window.location.href = isITLessEnv ? 'https://redhatgov.servicenowservices.com/css' : 'https://access.redhat.com/support'),
```

Comment:
The `isITLessEnv` is a boolean variable. This condition will not work.

Reply:
is this change better?